UsingSimpleSqlProvider/app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion compileSdkVer
+    buildToolsVersion buildToolsVer
+    defaultConfig {
+        applicationId "me.gurpreetsk.usingsimplesqlprovider"
+        minSdkVersion 15
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+
+    // Testing-only dependencies
+    testCompile "junit:junit:$junitVer"
+    androidTestCompile "com.android.support.test:runner:$androidTestRunnerVer"
+    androidTestCompile("com.android.support.test.espresso:espresso-core:$espressoCoreVer", {
```

Comment:
This is old. Use the latest one which does not require you to exclude any transitive dependency.